configure.ac

```diff
@@ -0,0 +1,29 @@
+AC_PREREQ([2.63])
+AC_INIT([ddclient], [3.9.1])
+AC_CONFIG_SRCDIR([ddclient])
+AC_CONFIG_AUX_DIR([build-aux])
+AC_CONFIG_MACRO_DIR([m4])
+AC_REQUIRE_AUX_FILE([tap-driver.sh])
+# If the automake dependency is bumped to v1.12 or newer, remove
+# build-aux/tap-driver.sh from the repository.  Automake 1.12+ comes
+# with tap-driver.sh, and autoreconf will copy in the version
+# distributed with automake.  (Automake 1.11 and older don't come with
+# tap-driver.sh, so build-aux/tap-driver.sh is checked in to keep the
+# above AC_REQUIRE_AUX_FILE line from causing configure to complain
+# about a mising file if the user has Automake 1.11.)
+AM_INIT_AUTOMAKE([1.11 -Wall -Werror foreign subdir-objects])
+AM_SILENT_RULES
+
+AX_WITH_PROG([PERL], perl)
+AX_PROG_PERL_VERSION([5.10.1], [],
+  [AC_MSG_ERROR([Perl 5.10.1 or newer not found])])
+m4_foreach_w([_module], [
+    Data::Validate::IP
```

Comment:
This needs an update after https://github.com/ddclient/ddclient/pull/178 got merged.

Reply:
Done.